bodyswap.lua

```diff
@@ -0,0 +1,255 @@
+-- Shifts player control over to another unit in adventure mode.
+-- author: Atomic Chicken
+-- based on "assumecontrol.lua" by maxthyme, as well as the defunct advtools plugin "adv-bodyswap"
+-- nemesis and historical figure creation uses slightly modified functions from modtools/create-unit
+
+--[====[
+
+bodyswap
+========
+
+Shifts player control over to another unit (including wild animals) in adventure mode.
+Usage:
+
+:bodyswap:
+    Swaps into the unit selected in the UI (for example, when viewing the unit's status screen or description).
+
+:bodyswap -unit [unitId]:
+    Swaps into the unit corresponding to the specified id.
+
+]====]
+
+local utils = require 'utils'
+validArgs = validArgs or utils.invert({
+'unit',
+'help'
+})
+local args = utils.processArgs({...}, validArgs)
+
+local usage = [====[

```

Comment:
I don't think two versions of the documentation is ideal. Sphinx should pick up anything between `[====[` and `]====]`, even if it's a string instead of a comment (see repeat, spawnunit, and lots of modtools scripts for examples).